cf-deployment.yml

```diff
@@ -1089,8 +1085,6 @@ instance_groups:
   vm_extensions:
   - diego-ssh-proxy-network-properties
   stemcell: default
-  update:
-    serial: true
```

Comment:
This `serial: true` here seems to ensures that the `scheduler` instance group happens after the `api` and `cc-worker` instance groups. This might have been required at some point in time due to jobs needing the `ccdb` to be in place, or possibly the `cloud_controller_ng` job to be running before one of the `scheduler` jobs was started.

I performed a deploy where I move the position of `scheduler` before the `api` instance group, but left `serial: true` enabled. The deploy was successful, showing that `scheduler` has no dependency on `api` or `cc-worker` instance groups and it should be safe to deploy them in parallel.